phidgets_temperature/src/temperature_ros_i.cpp

```diff
@@ -63,6 +63,20 @@ TemperatureRosI::TemperatureRosI(const rclcpp::NodeOptions& options)
         throw std::runtime_error("Publish rate must be <= 1000");
     }
 
+    this->declare_parameter("server_name",
+                            rclcpp::ParameterType::PARAMETER_STRING);
+    this->declare_parameter("server_ip",
+                            rclcpp::ParameterType::PARAMETER_STRING);
+    if (this->get_parameter("server_name", server_name_) &&
+        this->get_parameter("server_ip", server_ip_))
```

Comment:
I must admit I don't know the first thing about either the VINT serial hub nor the VINT network hub. I don't have either device for testing; that's what's making this so challenging even though it's really simple code.

In this PR, I've just ported the working code from #127 to ROS2. I haven't changed anything except replacing the ROS1 API calls with their ROS2 equivalents. Since we have this code in Noetic for a while now, and nobody has complained, I expect it's working.

You're definitely right about the RCLCPP_INFO call, but I believe somebody with access to a device for testing should make those changes.